config/consensus.go

```diff
@@ -496,7 +496,7 @@ func initConsensusProtocols() {
 
 		MaxBalLookback: 320,
 
-		MaxTxGroupSize:               1,
+		MaxTxGroupSize: 1,
```

Comment:
could you undo all unintended changes ? they won't contribute anything for supporting a single go version.
( if you really think we should make these changes, move these to a separate PR )

Reply:
Reverted unintended formatting.